src/main/java/com/fruitshop/servicesimpl/ResultMessageImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.fruitshop.servicesimpl;
+
+import com.fruitshop.dao.DataBaseManagerImpl;
+import com.fruitshop.model.Fruit;
+import com.fruitshop.services.ResultMessage;
+import java.util.Map;
+
+public class ResultMessageImpl implements ResultMessage {
+    public static final String COMMA = ",";
+    public static final String TITLE_TO_FILE = "fruit,quantity";
+    private final DataBaseManagerImpl dataBaseManagerImpl = new DataBaseManagerImpl();
```

Comment:
It is a good practice to inject service objects using a constructor. Any class must not pe responsible for creating its dependencies